starknet/src/interfaces/i_account.cairo

```diff
@@ -15,18 +15,3 @@ trait AccountABI<TState> {
     fn is_valid_signature(self: @TState, hash: felt252, signature: Array<felt252>) -> felt252;
```

Comment:
could we use the OZ library for the account interface? Or maybe there's a SNIP library somewhere we can use?

Reply:
yep imported from oz